### PR TITLE
Update readme for how to update code snippet to load sdk v2.

### PR DESCRIPTION
**Pull request by jpiyali**
Update readme for how to update code snippet to load sdk v2.
Update AISKU build to also build the snippet minified.

### CODE REVIEW

File: AISKU/README.md [hunk 1]
```diff
@@ -152,15 +155,33 @@ Most configuration fields are named such that they can be defaulted to falsey. A
...
 
 For a lightweight experience, you can instead install the basic version of Application Insights
 ```
 npm i --save @microsoft/applicationinsights-web-basic
 ```
-This version comes with the bare minimum amount of features and functionalities and relies on you to build it up as you see fit. For example, it performs no auto-collection (uncaught exceptions, ajax, etc). The APIs to send certain telemetry types, like `trackTrace`, `trackEvent`, etc, are not included in this version, so you will need to provide your own wrapper.
+This version comes with the bare minimum amount of features and functionalities and relies on you to build it up as you see fit. For example, it performs no auto-collection (uncaught exceptions, ajax, etc). The APIs to send certain telemetry types, like `trackTrace`, `trackException`, etc, are not included in this version, so you will need to provide your own wrapper. The only api that is available is `track`.
+
+
+## Application Insights backward compatibility validation
```
Comment (markwolff) [inline AISKU/README.md]: This can just be Backwards Compatibility? or maybe "Upgrading to New Version"

File: AISKU/README.md [hunk 2]
```diff
@@ -152,15 +155,33 @@ Most configuration fields are named such that they can be defaulted to falsey. A
...
+Telemetry envelope has some changes due to data schema updates.
+
+If you are using the current application insights PRODUCTION SDK (1.0.20) and want to see if the new SDK works in runtime, please update URL depending on your current SDK loading scenario:
+
+a) Download via CDN scenario:
+Update code snippet that you currently use to point to the following URL:
+"https://az416426.vo.msecnd.net/beta/ai.1.min.js"
+
+b) NPM scenario:
+Call downloadAndSetup to download full ApplicationInsights script from CDN and initialize it with instrumentation key
```
Comment (markwolff) [inline AISKU/README.md]: Use code format (backticks)

File: AISKU/README.md [hunk 3]
```diff
@@ -152,15 +155,33 @@ Most configuration fields are named such that they can be defaulted to falsey. A
...
+
+
+## Application Insights backward compatibility validation
+Breaking changes in the SDK in V2:
+To allow for better api signatures, some of the apis such as trackPageView, trackException have been updated. Running in IE8 or lower versions of the browser is not supported.
+Telemetry envelope has some changes due to data schema updates.
+
+If you are using the current application insights PRODUCTION SDK (1.0.20) and want to see if the new SDK works in runtime, please update URL depending on your current SDK loading scenario:
+
+a) Download via CDN scenario:
```
Comment (markwolff) [inline AISKU/README.md]: use markdown numbered list, or bold the **a)**, `**a)**`

File: AISKU/README.md (hunk 1 above)
Comment (jpiyali) [reply AISKU/README.md]: Good idea. done.

File: AISKU/README.md [hunk 4]
```diff
@@ -166,20 +166,25 @@ npm i --save @microsoft/applicationinsights-web-basic
...
+	```
+	"https://az416426.vo.msecnd.net/beta/ai.1.min.js"
+	```
 
-b) NPM scenario:
-Call downloadAndSetup to download full ApplicationInsights script from CDN and initialize it with instrumentation key
-AppInsights.downloadAndSetup({ instrumentationKey: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxx", url:  "https://az416426.vo.msecnd.net/beta/ai.1.min.js" });
+**b)** NPM scenario:
+	Call downloadAndSetup to download full ApplicationInsights script from CDN and initialize it with instrumentation key.  
+	appInsights.downloadAndSetup({
```
Comment (markwolff) [inline AISKU/README.md]: surround whole block with the tripleticks

**Pull request merged**